src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,35 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int operatingWeight;
+    private int bladeWidth;
```

Comment:
The field 'bladeWidth' should have an access modifier to encapsulate the field properly. As with 'operatingWeight', choose the appropriate access level for this field.